crates/tiny/src/ui.rs

```diff
@@ -172,15 +174,18 @@ pub(crate) fn send_msg(
     msg: String,
     is_action: bool,
 ) {
-    if src.serv_name() == "mentions" {
+    if src.serv_name().is_none() {
         if clients.is_empty() {
             ui.add_client_err_msg(
                 "No connected server found, please use `/connect <server>` to connect to a server",
                 &MsgTarget::CurrentTab,
             );
         } else {
             ui.add_client_err_msg(
-                "You are on the mentions tab, please use `/switch <tab name>` to switch to a tab",
+                &format!(
+                    "You are on the {} tab, please switch to a channel tab to send a message (`/switch <tab name>`)",
```

Comment:
You can send a message in server and user tabs too. I suggest something like

> You are on the {} tab, please switch to a server, channel, or user tab to send a message

I know this is not part of this PR, but I would also remove `/switch ...`. It's often easier to switch using `C-n`/`C-p`, I think suggesting `/switch` but not those is a bit weird. However, the user may have mapped next-tab/prev-tab to another key, so if we want to mention those keys too we should look next-tab/prev-tab in the keymap and suggest accordingly. It's simpler to just not say how to do it.

My approach is: if we can't make it perfect then often it's better to avoid than doing it poorly.

Reply:
Agreed.